tmt/base.py

```diff
@@ -230,6 +230,21 @@ def _lint_summary(self):
             verdict(None, "summary should not exceed 50 characters")
         return True
 
+    def has_link(self, target):
+        """ Whether object contains specified link """
+        if isinstance(target, Link):
+            link = target.get()
+        else:
+            link_parts = target.split(':', maxsplit=1)
+            if len(link_parts) == 1:
+                link = Link(link_parts[0]).get()[0]
+            else:
+                link = Link({link_parts[0]: link_parts[1]}).get()[0]
+        for candidate in self._metadata.get('link', []):
```

Comment:
Why not using directly the `self.link` attribute here?

Reply:
No idea. Fixed.